.styleci.yml

```diff
@@ -9,3 +9,7 @@ enabled:
   - ordered_use
   - strict
   - php_unit_construct
+
+disabled:
+  - single_line_class_definition
```

Comment:
How is this related to the Symfony 5 update?

Reply:
right. did that separately in #344 now